source/_integrations/unifiprotect.markdown

```diff
@@ -74,11 +76,27 @@ All known UniFi Protect devices are should be supported. Each UniFi Protect devi
 Each UniFi Protect camera will get the following entities added:
 
 * **Camera** - A camera entity for each camera channel and RTSP(S) combination found for each camera (up to 6). Only the highest resolution RTSPS camera entity will be enabled by default.
+* **Media Player** - If your camera has a speaker, you will get a media player entity that allows you to play audio to your camera's speaker. Any audio file URI that is playable by FFmpeg will be able to be played to your speaker, including via the [TTS Say Service](https://www.home-assistant.io/integrations/tts/#service-say).
```

Comment:
Don't use full URLs. 

Please adjust this to a link from the root of the domain.